pybamm/input/parameters/lithium-ion/electrolytes/lipf6_Prada2013/README.md

```diff
@@ -0,0 +1,7 @@
+# LiPF6 electrolyte parameters
```

Comment:
It seems that the electrolyte parameters for Prada2013 is basically Nyman 2008. If that is the case, instead of duplicating the files we should just define the parameter set `Prada2013` to take `Nyman2008` as electrolyte (seems to be what have been done anyway).